app/controllers/internal/privileged_reactions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Internal::PrivilegedReactionsController < Internal::ApplicationController
+  layout "internal"
+
+  PRIVILEGED_REACTION_CATEGORIES = %i[thumbsup thumbsdown vomit].freeze
```

Comment:
We should move tihs in the model and use it everywhere, same as `ReactionsController::MODERATION_CATEGORIES`

in another PR though :D